### PR TITLE
Get contents of mix.exs for a directory passed into the CLI

### PR DESCRIPTION
**Pull request by richardbatty**
Gets the dependencies of a passed-in elixir project

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **designed for change**, **easy to understand** and **able to cope when
things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [x] Description explains what you have done and why.
- [x] Tests have been written (or you have explained why you don't need
  tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

**Comment by jonnyarnold** [comment]
I think we can use [Mix.Project.get!/0](http://elixir-lang.org/docs/v1.0/mix/Mix.Project.html#get!/0) as long as we are in the folder for a project. This is a guess from the documentation though.

We might get a nice list of dependencies from this too.

**Comment by richardbatty** [comment]
Pinging the Tsar of Elixir @lpil

**Comment by lpil** [comment]
What's this now?

**Comment by jonnyarnold** [comment]
Oh mighty Tsar of Elixir, we come to you for guidance. And a code review.

There's not a lot here, but it would be good to get your input (and maybe involvement?) in an open-source project review system for Elixir.

File: lib/openreevoo.ex [hunk 1]
```diff
@@ -1,5 +1,24 @@
 defmodule Openreevoo do
   def main(args) do
-    IO.puts "hello world"
+    args |> directory |> in_directory(&deps/0)
```
Comment (lpil) [inline lib/openreevoo.ex]:
``` elixir
args |> directory |> File.cd!(&deps/0)
```

?

File: test/openreevoo_test.exs [hunk 2]
```diff
@@ -1,7 +1,40 @@
 defmodule OpenreevooTest do
   use ExUnit.Case
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  @tag :skip
+  test "The application asks the user to log in if they aren't already" do
   end
```
Comment (lpil) [inline test/openreevoo_test.exs:6]: If you're using Elixir 1.1 you can do this:

``` elixir
test "The application asks the user to log in if they aren't already"
```

And then you don't need to tag with `:skip`

File: test_application/mix.exs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  #   {:mydep, "~> 0.3.0"}
+  #
+  # Or git/path repositories:
+  #
+  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
+  #
+  # Type `mix help deps` for more examples and options
+  defp deps do
+    [
+      {:plug, ">= 0.4.0"}
```
Comment (lpil) [inline test_application/mix.exs]: Dogma? ;D

Comment (lpil) [inline test_application/mix.exs:31]: What's plug for?

File: lib/openreevoo.ex [hunk 4]
```diff
@@ -1,5 +1,24 @@
 defmodule Openreevoo do
```
Comment (lpil) [inline lib/openreevoo.ex:1]: Is this one word? OpenReevoo vs Openreevoo

**Comment by lpil** [comment]
If this is to be a website, I vote we use Phoenix.

**Comment by jonnyarnold** [comment]
The server will be a website, although we haven't decided what we'll be using to serve it...

File: test_application/mix.exs (hunk 3 above)
Comment (jonnyarnold) [inline test_application/mix.exs:31]: This is an example dependency for the test application.

File: lib/openreevoo.ex (hunk 4 above)
Comment (jonnyarnold) [inline lib/openreevoo.ex:1]: It doesn't matter right now.

File: test_application/mix.exs (hunk 3 above)
Comment (lpil) [inline test_application/mix.exs:31]: Oh, I see. Thanks.

**Pull request merged**